city_scrapers/spiders/chi_ssa_27.py

```diff
@@ -0,0 +1,123 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa27Spider(CityScrapersSpider):
+    name = "chi_ssa_27"
+    agency = "Chicago Special Service Area #27 Lakeview West"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.lakeviewchamber.com"]
+    start_urls = ["https://www.lakeviewchamber.com/ssa27"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        meeting = None
+        meet_path = "/html/body/div/div[2]/main/div/div[2]/div[2]" \
```

Comment:
A simpler way of doing this might be (with CSS selectors) `div[pageareaid="Sidebar"] li:first-child p`. We should iterate through the `p` tags instead of the text attribute because we'll want to parse the agenda links from them too.

We also want to get the separate committee meetings and minutes as well. You can get committee meetings and descriptions with `div[pageareaid="2"] li:last-child p` and minutes with `div[pageareaid="Sidebar"] li:last-child a`